R/missing_package_linter.R

```diff
@@ -1,6 +1,19 @@
 #' Missing package linter
 #'
-#' Check for missing packages in `library()`, `require()`, `loadNamespace()` and `requireNamespace()` calls.
+#' Check for missing packages in `library()`, `require()`, `loadNamespace()`, and `requireNamespace()` calls.
+#'
+#' @examples
+#' # will produce lints
+#' lint(
+#'   text = "library(statts)",
```

Comment:
I worry statts is too close to something that could conceivably be a real package name (if not now, then eventually). maybe pick something more obscure?

Reply:
Can't think of something that I don't see being a package name, so chose `xyzxyz`. Happy to change if you can think of something better.

Reply:
SGTM, I think a gibberish name is OK too...